src/MSBuild.UnitTests/PerfLog_Tests.cs

```diff
@@ -74,7 +74,7 @@ public void TestPerfLogEnabledProducedLogFile()
         }
 
         [Fact]
-        public void TestPerfLogDirectoryDoesNotExist()
+        public void TestPerfLogDirectoryGetsCreated()
```

Comment:
Why did this test used to exist? Seems a little odd to validate that we don't actually capture logs...

Reply:
The test validated that if you set a path that didn't exist, nothing would happen. I changed the logic of the eventlistener to create the directory, so the test wasn't necessary anymore.